pkg/language/chroma.go

```diff
@@ -42,6 +42,9 @@ import (
 	"github.com/danwakefield/fnmatch"
 )
 
+// Max file size supporting reading from file. Default is 512Kb.
+const maxFileSize = 512000
```

Comment:
Since this constant is repeated in a few places should we move it into a common place and import it everywhere?

Reply:
in Go is not recommended to create packages like utils, helpers as for other programming languages. As both values can change time by time I suggest to keep them as is.